tools/releasing/packaging/docker/nightly.dockerfile

```diff
@@ -0,0 +1,20 @@
+# Dockerfile to build a ubuntu image containing the installed develop version of preCICE
+
+FROM ubuntu:22.04
```

Comment:
I like the fact that we now use a standard image instead of a customized one :+1:

Reply:
> I think if we always stick to the baseline version of the preCICE release this is a simple rule to follow and also more clear from the outside perspective. Our baseline for preCICE v3 is ubuntu 22.04 (?).

We support the latest 2 Ubuntu LTS versions, meaning our baseline is 20.04 and in April we will switch to 22.04.

> When do we change ubuntu:22.04 to something newer?

Every 2 years with the LTS releases, aka when baselines shift.

> Is a non-LTS also reasonable for a nightly build?

I would say no as this would unnecessarily restrict the support of past releases.

We again have conflicting interests.

Using the baseline Ubuntu to build release images makes sense from a testing perspective as you make sure that the baseline works.
Use the latest LTS to build release images makes sense from a production perspective as you make sure that the newest packages are available.

I'll open an issue about this.

Reply:
There you go https://github.com/precice/precice/issues/1950

Reply:
I'll mark this as resolved, as this PR currently uses the same version we already used for the releases.

Reply:
Sounds good. Which ubuntu we use in the end is a side-track that we can resolve in #1950 let's focus on the main issue here.